pkg/render/intrusiondetection/dpi/dpi_test.go

```diff
@@ -208,6 +210,12 @@ var _ = Describe("DPI rendering tests", func() {
 		dpiCertSecret, err = certificateManager.GetOrCreateKeyPair(cli, render.DPITLSSecretName, common.OperatorNamespace(), []string{""})
 		Expect(err).NotTo(HaveOccurred())
 
+		snortRules, err := embed.AsConfigMap(
+			applicationlayer.ModSecurityRulesetConfigMapName,
```

Comment:
should this be `dpi.DeepPacketInspectionSnortRulesCmName`?

Reply:
That's right , this is a copy-paste error. Corrected.